airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/AbstractJdbcDestination.java

```diff
@@ -113,4 +111,10 @@ public AirbyteMessageConsumer getConsumer(JsonNode config, ConfiguredAirbyteCata
     return JdbcBufferedConsumerFactory.create(getDatabase(config), sqlOperations, namingResolver, config, catalog);
   }
 
+  public static ConnectorSpecification getSpec() throws IOException {
```

Comment:
what about doing what we do on the source side and just having either `RedshiftDestination` or `SwitchingDestination` or `AbstractJdbcDestination` extend `BaseConnector`? Keep it all the same?

e.g.
```
public abstract class AbstractJdbcSource extends BaseConnector implements Source 
```